### PR TITLE
Fix label for retrieving metrics

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

### CODE REVIEW

File: thoth/metrics_exporter/jobs/security.py [hunk 1]
```diff
@@ -54,8 +54,9 @@ def get_si_unanalyzed_python_packages_versions(cls) -> None:
 
         metric_name = "thoth_graphdb_total_number_si_unanalyzed_python_packages"
         metric = Configuration.PROM.get_current_metric_value(
-            metric_name=metric_name, label_config={"instance": cls._METRICS_EXPORTER_INSTANCE}
+            metric_name=metric_name, label_config={'field': cls._METRICS_EXPORTER_INSTANCE}
         )
+        print(metric)
```
Comment (harshad16) [inline thoth/metrics_exporter/jobs/security.py:59]: is this intentional?

**Review by harshad16 — CHANGES_REQUESTED** [review]
lgtm
need to fix the pre-commit issue.

File: thoth/metrics_exporter/jobs/security.py (hunk 1 above)
Comment (pacospace) [reply thoth/metrics_exporter/jobs/security.py:59]: nice catch, I was testing the metric :) thanks!

**Review by fridex — APPROVED** [review]
lgtm 👍🏻

**Pull request merged**